authorization-service/serverless.ts

```diff
@@ -0,0 +1,56 @@
+import type { AWS } from "@serverless/typescript";
+
+import { protectedFn, basicAuthorizer } from "./src/functions";
+
+const serverlessConfiguration: AWS = {
+  service: "authorization-service",
+  frameworkVersion: "3",
+  plugins: ["serverless-esbuild", "serverless-offline"],
+  provider: {
+    name: "aws",
+    runtime: "nodejs14.x",
+    stage: "dev",
+    region: "eu-west-1",
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: "1",
+      NODE_OPTIONS: "--enable-source-maps --stack-trace-limit=1000",
+      nataliagulko: "TEST_PASSWORD",
```

Comment:
minor: private information should not be visible in code

reference from the task:
Do not send your credentials to the GitHub. Use .env file and serverless-dotenv-plugin serverless plugin to add environment variables to the lambda. Add .env file to .gitignore file.

Reply:
Oh, yes, sure. I think, I forgot to remove it! 😱
I have this credentials at .env as well

Thank you!